src/plugins/plugin-mapbox-gl/custom-layers/bitmap-layer/bitmap-layer-fragment.js

```diff
@@ -9,32 +9,33 @@ uniform sampler2D bitmapTexture;
 
```

Comment:
Let remove this layer if it is no longer being used.

Reply:
It was created for the first approach of video tiles, but as we are going to use the second approach (even if that means that we are going to fork a version of mapbox) I will remove all the dependencies

Reply:
Try to avoid forks, it means more to maintenance

Reply:
We can help the guy creating a PR to Mapbox, meanwhile, there is no other option...